src/devices/Cargo.toml

```diff
@@ -34,6 +34,8 @@ utils = { path = "../utils" }
 polly = { path = "../polly" }
 rutabaga_gfx = { path = "../rutabaga_gfx", features = ["virgl_renderer", "virgl_renderer_next"], optional = true }
 
+imago = { git = "https://gitlab.com/hreitz/imago.git", branch = "main", features = ["sync-wrappers", "vm-memory"] }
```

Comment:
@XanClic are you planning on releasing a 0.1.0 of `imago`?

Reply:
Yes, right. I will do a release. Do we need anything except me declaring 0.1.0 done and pushing it to crates.io?

Reply:
I’ve just pushed 0.1.0 to crates.io.

Reply:
Thank you! AFAIK that's all we need.